gitlab.sh

```diff
@@ -1,28 +0,0 @@
-#!/bin/bash
```

Comment:
The old gitlab.sh and travis.sh were removed, and two symbolic links were created pointing to ci.sh for backward compatibility.